test/unit/ClauseResultsBuilder.test.ts

```diff
@@ -349,6 +350,21 @@ describe('ClauseResultsBuilder', () => {
       const arrayObject = { array: [1, 2, 3] };
       expect(ClauseResultsBuilder.prettyResult(arrayObject)).toEqual('{\n  array: [1,\n         2,\n         3]\n}');
     });
+
+    test('should create pretty code', () => {
+      const code = new Code('testCode', 'http://snomed.info/sct');
+      expect(ClauseResultsBuilder.prettyResult(code)).toEqual('CODE: SNOMEDCT testCode');
+    });
+
+    test('should create pretty code for unmapped system URL', () => {
+      const code = new Code('testCode', 'www.example.com');
+      expect(ClauseResultsBuilder.prettyResult(code)).toEqual('CODE: www.example.com testCode');
+    });
+
+    test('should create pretty code for undefined system URL', () => {
```

Comment:
Along with my above comment on integration tests, is there any way we can expand these unit tests more to cover what that integration test is covering instead?

Reply:
I played around with trying to do unit testing with FHIRObjects a couple of times, but because that class is not actually exported, we run into a lot of trouble, and I think would have to mock out everything to the point of non-usefulness.